permissions/plugin-sealights.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "sealights-plugin"
+github: "jenkinsci/sealights-plugin"
+paths:
+- "org/jenkins-ci/plugins/sealights-plugin"
```

Comment:
The path is actually the `groupId` (https://github.com/jenkinsci/sealights-plugin/blob/master/pom.xml#L14) from your pom.xml with periods replaced with forward slashes and then the `artifactId` (https://github.com/jenkinsci/sealights-plugin/blob/master/pom.xml#L15).

So, in your case, it would need to be `io/sealights/plugins/sealights`

Reply:
Thanks for the feedback.
Fixed.